reactor-netty-core/src/main/java/reactor/netty/tcp/TcpResources.java

```diff
@@ -165,25 +165,58 @@ public boolean daemon() {
 	}
 
 	/**
-	 * Use {@link #disposeLoopsAndConnections()}
+	 * {@link TcpResources} provides the global TCP resources, as such {@link TcpResources} lifecycle is
+	 * longer than regular {@link LoopResources} and {@link ConnectionProvider} implementations lifecycle.
+	 * In order to prevent an unintended disposal of the global TCP resources, {@link #disposeLoopsAndConnections()}
+	 * is exposed and should be used.
+	 * <p>
+	 * This method has {@code NOOP} implementation.
```

Comment:
The order in which this is explained is slightly confusing to me in the sense that if the method is a no-op why would it cause any issues. Perhaps something along these lines:

"This has is a no-op by default in order to prevent unintended disposal of the global TCP resources which has a longer lifecycle than regular LoopResources and ConnectionProvider. If you do mean to shut global resources, please use `disposeLoopsAndConnections` instead."

Reply:
Please take a look now